toqito/state_ops/tests/test_renyi_entropy.py

```diff
@@ -0,0 +1,40 @@
+"""Test Renyi Entropy."""
+
+import numpy as np
+import pytest
+
+from toqito.state_ops import renyi_entropy
+
+# Example density matrices for testing
```

Comment:
Can you please use `pytest.parameterize` to cycle over these different cases and consolidate the tests? You can look at the other tests that use `parameterize` in toqito for reference.